contrib/gitian-keys/keys.txt

```diff
@@ -1,41 +1,51 @@
 9D3CC86A72F8494342EA5FD10A41BDC3F4FAFF1C Aaron Clauson (sipsorcery)
 617C90010B3BD370B0AC7D424BB42E31C79111B8 Akira Takizawa
 E944AE667CF960B1004BC32FCA662BE18B877A60 Andreas Schildbach
-152812300785C96444D3334D17565732E08E5E41 Andrew Chow
+152812300785C96444D3334D17565732E08E5E41 Andrew Chow (achow101)
 590B7292695AFFA5B672CBB2E13FC145CD3F4304 Antoine Poinsot (darosior)
 0AD83877C1F0CD1EE9BD660AD7CC770B81FD22A8 Ben Carman (benthecarman)
 912FD3228387123DC97E0E57D5566241A0295FA9 BtcDrak
 C519EBCF3B926298946783EFF6430754120EC2F4 Christian Decker (cdecker)
+18AE2F798E0D239755DA4FD24B79F986CBDF8736 Chun Kuan Le (ken2812221)
 F20F56EF6A067F70E8A5C99FFF95FAA971697405 centaur
-C060A6635913D98A3587D7DB1C2491FFEB0EF770 Cory Fields
+C060A6635913D98A3587D7DB1C2491FFEB0EF770 Cory Fields (cfields)
 BF6273FAEF7CC0BA1F562E50989F6B3048A116B5 Dev Random
-6D3170C1DC2C6FD0AEEBCA6743811D1A26623924 Douglas Roark
+6D3170C1DC2C6FD0AEEBCA6743811D1A26623924 Douglas Roark (droark)
+1C6621605EC50319C463D56C7F81D87985D61612 Emanuele Cisbani (cisba)
 9A1689B60D1B3CCE9262307A2F40A9BF167FBA47 Erik Mossberg (erkmos)
 D35176BE9264832E4ACA8986BF0792FBE95DC863 fivepiece
+6F993B250557E7B016ADE5713BDCDA2D87A881D9 Fuzzbawls (Fuzzbawls)
```

Comment:
seems redundant

Reply:
It's not. The script ignores the "real name" (the first part) only looks at the aliases in parentheses.